proposals/NNNN-atf_security_proposal.md

```diff
@@ -0,0 +1,135 @@
+# Security ATF feature
+
+* Proposal: [SDL-NNNN](NNNN-atf_security_proposal.md)
+* Author: [Alexander Kutsan](https://github.com/LuxoftAKutsan)
+* Status: **Awaiting review**
+* Impacted Platforms: [ATF]
+
+## Introduction
+
+This proposal describes support of security sessions in ATF (Automated Test Framework).
+ATF should be able to test SDL security feature.
+
+Main ATF features for checking SDL security are:
+ - Support of secure SDL sessions;
+ - Support of testing broken handshake;
+ - Support of sending raw data in secure channel;
+ - Support of expected secure requests\responses\notifications;
+ - Support Transport Layer Security (TLS) and Datagram Transport Layer Security (DTLS). 
+
+## Motivation
+
+Motivation of this proposal is to create ability of coverage the secure sessions. Which is the most important not covered part of SDL functionality.
+
+Manual testing of security SDL feature is slow and expensive. Also there is a big probability of error during manual testing.
+Testing automatization of all SDL use cases is the best option to be sure that new code does not break SDL functionality.
+
+ATF has possibility to cover almost all SDL use cases with automatic testing. But there is still some functionality that remains uncovered my ATF:

```

Comment:
Please update to "ATF has possibility to cover almost all SDL use cases with automatic testing, but there is still some functionality that remains uncovered by ATF:"